resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -584,6 +595,128 @@ class TextCursor
     }
 }
 
+[DSCResource()]
+class ColorFilter
+{
+    # Key required. Do not set.
+    [DscProperty(Key)] [string] $SID
+    [DscProperty()] [nullable[bool]] $FilterStatus
+    [DscProperty()] [ColorFilters] $FilterColor
+    [DscProperty()] [nullable[bool]] $KeyboardShortcutStatus
+
+    static hidden [string] $FilterStatusProperty = 'Configuration'
+    static hidden [string] $FilterStatusActiveProperty = 'Active'
+    static hidden [string] $FilterStatusValue = 'colorfiltering'

```

Comment:
I'm not sure why this is needed

Reply:
My mistake for being unclear - unclear in words or unclear in code, I'm not sure, probably both.